### ISSUE TITLE
Record performance metrics for accessibility tests in DataDog.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#11629]
## Description
Each of the accessibility tests should receive a corresponding gauge metric for elapsed time in DataDog.  This should be reported only on staging.

## Acceptance Criteria
- [ ] Elapsed time is recorded for each accessibility test.
- [ ] That figure is reported upstream to DataDog with an appropriate name in a form such that it can be queried for and exposed in dashboards.
